tests/trading/test_orders.py

```diff
@@ -23,207 +29,215 @@
 WORSE_ORDER_ID = 6
 GAS_PRICE = 7
 
+ZEROED_ORDER_ID = '\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
+
 def test_walkOrderList_bids(contractsFixture):
     market = contractsFixture.binaryMarket
-    orders = contractsFixture.contracts['orders']
-    ordersFetcher = contractsFixture.contracts['ordersFetcher']
+    orders = contractsFixture.contracts['NewOrders']
+    ordersFetcher = contractsFixture.contracts['NewOrdersFetcher']
     outcomeID = 1
     order = {
         "orderID": 5,
-        "type": 1,
+        "type": BID,
         "fxpAmount": fix('1'),
         "fxpPrice": fix('0.6'),
         "sender": tester.a0,
         "outcome": outcomeID,
         "fxpMoneyEscrowed": fix('0.6'),
         "fxpSharesEscrowed": 0,
-        "betterOrderID": 0,
-        "worseOrderID": 0,
+        "betterOrderID": HASHED_ORDER_ID_0,
+        "worseOrderID": HASHED_ORDER_ID_0,
         "tradeGroupID": 0
     }
-    assert(orders.saveOrder(order["orderID"], order["type"], market.address, order["fxpAmount"], order["fxpPrice"], order["sender"], order["outcome"], order["fxpMoneyEscrowed"], order["fxpSharesEscrowed"], order["betterOrderID"], order["worseOrderID"], order["tradeGroupID"], 1) == 1), "Save order"
-    bestOrderID = int(orders.getBestOrderID(1, market.address, outcomeID), 16)
-    worstOrderID = int(orders.getWorstOrderID(1, market.address, outcomeID), 16)
-    assert(bestOrderID == 5)
-    assert(worstOrderID == 5)
+    hashedOrderID5 = orders.saveOrder(HASHED_ORDER_ID_0, order["type"], market.address, order["fxpAmount"], order["fxpPrice"], order["sender"], order["outcome"], order["fxpMoneyEscrowed"], order["fxpSharesEscrowed"], order["betterOrderID"], order["worseOrderID"], order["tradeGroupID"])
```

Comment:
Recommend naming this just `orderId`.  Order IDs are always 20-byte arrays whose provenance is a hash but that isn't relevant to the reader.